### PR TITLE
Adding some lints, and minor cleanup

### PR DESCRIPTION
**Pull request by domesticmouse**

### CODE REVIEW

File: shrine/lib/colors.dart [hunk 1]
```diff
@@ -14,15 +14,15 @@
 
 import 'package:flutter/material.dart';
 
-const kShrinePink50 = const Color(0xFFFEEAE6);
-const kShrinePink100 = const Color(0xFFFEDBD0);
-const kShrinePink300 = const Color(0xFFFBB8AC);
-const kShrinePink400 = const Color(0xFFEAA4A4);
+const kShrinePink50 = Color(0xFFFEEAE6);
```
Comment (RedBrogdon) [inline shrine/lib/colors.dart]: Will Dart still infer these to be constants?

File: veggieseasons/lib/widgets/settings_group.dart [hunk 2]
```diff
@@ -69,7 +69,7 @@ class SettingsGroup extends StatelessWidget {
     this.header,
     this.footer,
   })  : assert(items != null),
-        assert(items.length > 0);
+        assert(items.isNotEmpty);
```
Comment (RedBrogdon) [inline veggieseasons/lib/widgets/settings_group.dart]: Do people use these properties (`isEmpty`, `isNotEmpty`) consistently? I have nothing more than personal taste to back up my opinion, but for some reason I don't like them.

File: shrine/lib/colors.dart (hunk 1 above)
Comment (domesticmouse) [reply shrine/lib/colors.dart]: The second `const` is not needed in Dart 2 IIUC, and this change is encouraged by the `unnecessary_const` lint check.

File: veggieseasons/lib/widgets/settings_group.dart (hunk 2 above)
Comment (domesticmouse) [reply veggieseasons/lib/widgets/settings_group.dart]: I prefer them as the code reads better, less decoding overhead. As to usage, the `prefer_is_empty` and `prefer_is_not_empty` lint checks will educate and encourage developers. This is why I'm on my add lint checks to everything spree, I witnessed how much value the Go community got out of having one blessed code style.

**Comment by willlarche** [comment]
@RedBrogdon , if this is not materially different from what's in the flutter gallery, we should consider removing it here and bringing it down to 2 places.

**Comment by domesticmouse** [comment]
Are we going to land this PR, or should I close it out?

**Pull request closed without merging**